tests/common/DotNet.cs

```diff
@@ -113,6 +113,35 @@ public static ExecutionResult AssertNew (string outputDirectory, string template
 			return new ExecutionResult (output, output, rv.ExitCode);
 		}
 
+		public static ExecutionResult InstallTool (string tool, string path)
+		{
+			var installed = ExecuteCommand (Executable, "tool", "list", "--tool-path", path);
+			if (!installed.StandardOutput.ToString ().Contains (tool))
+				ExecuteCommand (Executable, "tool", "install", tool, "--tool-path", path);
+			return installed;
```

Comment:
I think you need to update the `installed` variable after installing the tool, otherwise this will only work if the tool is already installed.

Reply:
good catch, updated the variable